app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils eutils versionator
+
+DESCRIPTION="Master PDF Editor is a complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+
+SRC_URI="http://get.code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
+	app-arch/bzip2
+	dev-libs/double-conversion
+	dev-libs/glib
+	dev-libs/icu
+	dev-libs/openssl
+	media-gfx/graphite2
+	media-gfx/sane-backends
+	media-libs/freetype
+	media-libs/harfbuzz
+	media-libs/libjpeg-turbo
+	media-libs/libpng
+	media-libs/tiff
+	x11-libs/libX11
+	x11-libs/libXau
+	x11-libs/libxcb
+	x11-libs/libXdmcp
+	x11-libs/libXext
+	>=dev-qt/qtsvg-5.4:5
+	>=dev-qt/qtnetwork-5.4:5
+	>=dev-qt/qtgui-5.4:5
+	>=dev-qt/qtprintsupport-5.4:5
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+S="${WORKDIR}/${PN}-$(get_major_version ${PV})"
+
+src_install() {
+	local dest=/opt/${PN}
+	local bin_name=masterpdfeditor4
```

Comment:
Please don't introduce unnecessary temporary variables, they cause readability to suffer for no apparent gain.